packages/manager/src/features/linodes/LinodesCreate/UserDataAccordion/UserDataAccordion.tsx

```diff
@@ -49,22 +51,31 @@ const UserDataAccordion = (props: Props) => {
     padding: `0px 24px 24px ${renderNotice ? 0 : 24}px`,
   };
 
+  const fromBackupOrFromLinode =
+    createType && ['fromBackup', 'fromLinode'].includes(createType);
+
   return (
     <Accordion
       heading={<UserDataAccordionHeading createType={createType} />}
-      style={{ marginTop: renderNotice && renderCheckbox ? 0 : 24 }} // for now, these props can be taken as an indicator we're in the Rebuild flow.
+      style={{
+        marginTop: renderNotice && renderCheckbox ? 0 : 24,
+      }} // for now, these props can be taken as an indicator we're in the Rebuild flow.
       headingProps={{
         variant: 'h2',
       }}
       summaryProps={{
         sx: { padding: '5px 24px 0px 24px' },
+        style: {
+          alignItems: fromBackupOrFromLinode ? 'flex-start' : 'center',
+        },
```

Comment:
Can the use of `sx` and `style` be consolidated?

```suggestion
        sx: {
          padding: '5px 24px 0px 24px',
          alignItems: fromBackupOrFromLinode ? 'flex-start' : 'center',
        },
```

Reply:
I agree with consolidating this